pages/content/amp-dev/documentation/guides-and-tutorials/develop/custom-javascript.md

```diff
@@ -39,17 +39,17 @@ The `amp-script` component sends messages between the Web Worker thread and the
 
 ## Custom scripts caching
 
-The [AMP cache](../../../documentation/guides-and-tutorials/learn/amp-caches-and-cors/how_amp_pages_are_cached.md) serves custom JavaScript files inserted with `<amp-script>` same as AMP component scripts. This ensures that any custom JavaScript will not slow the speed of an AMP document.
+The [AMP cache](../../../documentation/guides-and-tutorials/learn/amp-caches-and-cors/how_amp_pages_are_cached.md) serves custom JavaScript files inserted with `<amp-script>` the same way it serves AMP component scripts, which ensures that any custom JavaScript will not slow the speed of an AMP document.
 
 The AMP cache proxies the JavaScript files and then delivers them. Users can expect the same performance experience from a page using `<amp-script>` as a page that doesn't include it.
 
 # Using `<amp-script>`
 
-To keep experiences on AMP pages consistent, limitations exist on `<amp-script>` to guarantee a fast loading time and smooth UI.
+To guarantee AMP pages will load consistently fast and with smooth UIs, limitations exist on `<amp-script>`.
 
 ## Initialization
 
-JavaScript inside the Web Worker allows minimal change to the DOM on load. Changes allowed during this phase are:
+JavaScript inside the Web Worker enables minimal change to the DOM on load. Changes allowed during this phase are:
```

Comment:
allows might be better here – we intentionally restrict what is possible.

Reply:
Done, thanks!